less/index.less

```diff
@@ -48,12 +48,204 @@ table {
 	border-spacing: 0;
 }
 
+/******************** General Code ********************/
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
+// Mixins
+.flexbox(){
+	display:flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+
+// Variables
+@fonttitles:'Sansita', sans-serif;
+@fontbody:'Pontano Sans', sans-serif;
+@main:2px solid red;
+@items:2px solid blue;
+@component:2px solid green;
+@margin: 0 auto;
+@indent: 5%;
+
 html, body {
-    height: 100%;
+	// height: 100%;
+	font-size: 62.5%;
+	// border: @main;
+	max-width: 90%;
+	margin: @margin;
+	background-image: linear-gradient(lightblue, lightpink);
+}
+
+h1, h2, h3, nav{
+	font-family: @fonttitles;
+	font-size:2rem;
+
+}
+
+p, table, button, li, label{
+	font-family:@fontbody;
+	font-size:1.5rem;
+}
+
+/******************** RESUME CODE ********************/
+
+nav{
+	// border:@main;
+	margin: @margin;
+	margin-top: 2%;
+	width:70%;
+	padding-bottom: 0.5%;
+	border-bottom: 1px solid black;
+	display: flex;
+	justify-content: space-around;
+	a{
+		// border: @items;
+		text-decoration: none;
+		color: black;
+	}
+}
+
+.introContainer{
+	// position:absolute;
+	// border: @main;
+	padding:4%;
+	width:60%;
+	// height:60%;
+	margin: @margin;
+	.flexbox();
+	border-bottom:1px dashed black;
+	
+	h1{
+		// border: @component;
+		width:100%;
+		margin:2% auto;
+		text-align: center;
+		font-size: 2.5rem;
+	}
+
+	button{
+		// padding:0.5%;
+		width: 30%;
+		// margin-left:38%;
+		border:1px solid black;
+		margin:@margin;
+		border-radius: 0.5rem;
+		background-color: rgba(255, 182, 193, 0.432);
+	}
+
+	.intro1{
+		// width:50%;
+		// margin: @margin;
+	}
+	.intro2{
+		// width: 50%;
+		// margin:@margin;
+		margin:@indent auto;
+		// color:blue;
+	}
+}
+
+.selfie{
+	// border:@component;
+	margin-left: @indent;
+	margin-top: 4%;
+	width: 30%;
```

Comment:
add a nested
```
img {
    width: 100%;
]
```
Now, the `<div>` tag will shrink as the page gets smaller, but the image will maintain it's proper aspect ratio 👍